extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -31,6 +31,8 @@ import JumpPresets from '../../utils/JumpPresets';
 const EVENTS = {
   VIEWPORT_DATA_CHANGED:
     'event::cornerstoneViewportService:viewportDataChanged',
+  VIEWPORT_STACK_SET: 'event::cornerstone::viewportservice:viewportstackset',
+  VIEWPORT_VOLUME_SET: 'event::cornerstone::viewportservice:viewportvolumeset',
```

Comment:
Why do you need these two events?
We already have `STACK_VIEWPORT_NEW_STACK` and `VOLUME_VIEWPORT_NEW_VOLUME` in cs3d